extensions/mypy_extensions.py

```diff
@@ -7,16 +7,78 @@
 
 # NOTE: This module must support Python 2.7 in addition to Python 3.x
 
+import sys
+import types
+from typing import _type_check  # type: ignore
 
-def TypedDict(typename, fields):
-    """TypedDict creates a dictionary type that expects all of its
+
+def _check_fails(cls, other):
+    try:
+        if sys._getframe(1).f_globals['__name__'] not in ['abc', 'functools']:
```

Comment:
Hm... If I comment out this line things still work, suggesting that it's not important that it returns False when invoked from those modules. I suppose this is from when this code was part of typing.py?